reascripts/common/libs/EnvUtil.lua

```diff
@@ -0,0 +1,24 @@
+--[[
+
+  EnvUtil.lua - utilities for querying REAPER environment
+
+]]--
+
+EnvUtil = {}
+
+EnvUtil.is_windows = function()
+  return EnvUtil._bool(reaper.GetOS():find('Win', 1, true))
+end
+
+EnvUtil.is_mac = function()
+  local is_mac = reaper.GetOS():find('OSX', 1, true)
+    or reaper.GetOS():find('macOS', 1, true)
+
+  return EnvUtil._bool(is_mac)
+end
+
+EnvUtil.is_linux = function()
+  return EnvUtil._bool(reaper.GetOS():find('Other'))
```

Comment:
Any reason why this call to find does not pass `1, true` like the other ones?

Reply:
if there was, i can't remember it 😅